deploy/platform/kubernetes/Makefile

```diff
@@ -22,6 +22,11 @@ include Makefile.in
 features := $(subst $(comma), ,$(FEATURE_FLAGS))
 features := $(foreach f,$(features),feature-$(f))
 
+BACKEND_SERVICE := oap
+ifneq (,$(findstring satellite,$(features)))
+	BACKEND_SERVICE := satellite
+endif
```

Comment:
2. `BACKEND_ADDRESS` doesn't contain the port, please make (1) and (2) consistent